geniza/templates/404.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
```

Comment:
is there enough common between 404/500 to merit a base error template?

Reply:
There's a lot in common, but I'm not sure a base template would actually reduce more than one or two lines of repetition. Let me know what you think after these new revisions!

Reply:
I agree, doesn't seem worth it to create a base error template.